resources/dicts/events/misc_events/general.json

```diff
@@ -1,5659 +1,2653 @@
+  "biome": ["any"],
+  "camp": ["any"],
+  "season": ["any"],
+  "tags": ["murder_reveal"],
+  "weight": 10,
+  "event_text": "r_c is stunned when {PRONOUN/r_c/subject} {VERB/r_c/discover/discovers} m_c carelessly bragging about killing mur_c to a wandering loner.",
+  "m_c": {
+    "trait": ["oblivious"]
+  },
+  "r_c": {
+    "age": ["adolescent", "young adult", "adult", "senior adult", "senior"]
+  }
+},
+{
+  "event_id": "gen_misc_murderreveal_any3",
+  "biome": ["any"],
+  "camp": ["any"],
+  "season": ["any"],
+  "tags": ["murder_reveal"],
+  "weight": 10,
+  "event_text": "m_c is truthful when r_c questions {PRONOUN/m_c/object} about mur_c's strange death. m_c claims that mur_c deserved to die after what {PRONOUN/mur_c/subject} did to m_c.",
+  "m_c": {
+    "trait": ["vengeful"]
+  },
+  "r_c": {
+    "age": ["adolescent", "young adult", "adult", "senior adult", "senior"]
+  }
+},
+{
+  "event_id": "gen_misc_murderreveal_any4",
+  "biome": ["any"],
+  "camp": ["any"],
+  "season": ["any"],
+  "tags": ["murder_reveal", "clan_discovery"],
+  "weight": 5,
+  "event_text": "m_c boldly confesses to the entire Clan that {PRONOUN/m_c/subject} killed mur_c. m_c claims that mur_c deserved to pay for what {PRONOUN/mur_c/subject} did.",
+  "m_c": {
+    "trait": ["vengeful"]
+  },
+  "relationships": [
+    {
+      "cats_from": ["clan"],
+      "cats_to": ["m_c"],
+      "mutual": false,
+      "values": ["dislike"],
+      "amount": 30
+    },
+    {
+      "cats_from": ["clan"],
+      "cats_to": ["m_c"],
+      "mutual": false,
+      "values": ["trust", "respect", "comfort"],
+      "amount": -30
+    }
+  ]
+},
+{
+  "event_id": "gen_misc_murderreveal_any5",
+  "biome": ["any"],
+  "camp": ["any"],
+  "season": ["any"],
+  "tags": ["murder_reveal", "clan_discovery"],
+  "weight": 10,
+  "event_text": "m_c's cold eyes don't blink when {PRONOUN/m_c/subject} {VERB/m_c/are/is} accused of murdering mur_c, and only simply states that the Clan is better off without mur_c anyways.",
+  "m_c": {
+    "trait": ["cold"]
+  }
+},
+{
+  "event_id": "gen_misc_murderreveal_any6",
+  "biome": ["any"],
+  "camp": ["any"],
+  "season": ["any"],
+  "tags": ["murder_reveal"],
+  "weight": 10,
+  "event_text": "m_c doesn't change expressions in the slightest when r_c accuses {PRONOUN/m_c/object} of murdering mur_c. Who cares?",
+  "m_c": {
+    "trait": ["cold"]
+  },
+  "r_c": {
+    "age": ["adolescent", "young adult", "adult", "senior adult", "senior"]
+  }
+},
+{
+  "event_id": "gen_misc_murderreveal_any7",
+  "biome": ["any"],
+  "camp": ["any"],
+  "season": ["any"],
+  "tags": ["murder_reveal"],
+  "weight": 5,
+  "event_text": "r_c avoids m_c at all costs after accidentally witnessing mur_c's murder. r_c knows {PRONOUN/r_c/subject} can't keep it a secret but is terrified of what m_c might do.",
+  "m_c": {
+    "trait": ["bloodthirsty"]
+  },
+  "r_c": {
+    "age": ["kitten","adolescent", "young adult", "adult", "senior adult", "senior"]
```

Comment:
let's remove the kitten and adolescent to help avoid cats who aren't old enough to have witnessed the murder from getting this event.  it'll still be possible, but hopefully not as likely.  i hope eventually we can add some sort of filter to help prevent this in general, but for now this is easier.

Reply:
Changed within [86bfd96](https://github.com/scribblecrumb/clangen/pull/27/commits/86bfd9648c8800dc11b2893afc7b55e64701738a), aswell as other events in the same vein